samples/throttling/sample.md

```diff
@@ -62,4 +62,4 @@ Handles the detection of `Octokit.RateLimitExceededException` and defers the mes
 
 snippet: ThrottlingBehavior
 
-NOTE: The above example creates a new message and does not copy headers of the original message. Should headers be required they need to be copied from the `context.Headers` into the  `SendOptions`.
+NOTE: The above example sends a copy of the original message, but does not copy the headers of the original message. If the headers of the original message are required, they must be copied from `context.Headers` to `SendOptions`.
```

Comment:
Does this note apply to the entire sample, or just the _The pipeline behavior_ section?

If the latter, where is the code that copies the message?

Reply:
The behavior only

Reply:
OK. Which bit of code does that? It may be worth pointing out in the note.

Reply:
Technically this part but this can get convoluted and complex very quickly

https://github.com/Particular/docs.particular.net/blob/master/samples/throttling/Core_7/Limited/ThrottlingBehavior.cs#L56

because from the perspective of the code we just pass in `MessageBeingHandled` and then you have to explain what copying means in this context.